src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { PropsWithChildren } from "react";
+import { Navbar } from "../Navbar/Navbar";
+import { Footer } from "../Footer/Footer";
+import Head from "next/head";
+import styles from "./styles.module.css";
+import { Alert } from "@aivenio/aquarium";
+import { useAppContext } from "@/context";
```

Comment:
NIT: Maybe we could include same sort linter config we have for Console for reordering and grouping imports.

Reply:
I'll create a follow-up action for setting up a Prettier config and using the Aiven ESlint config. Will do that either separately later or in this PR.